vortex-array/src/array/display/mod.rs

```diff
@@ -12,40 +12,10 @@ use vortex_error::VortexExpect as _;
 
 use crate::Array;
 
-/// A shim used to display the logical contents of an array.
-///
-/// For example, an `i16` typed array containing the first five non-negative integers is displayed
-/// as: `[0i16, 1i16, 2i16, 3i16, 4i16]`.
-///
-/// # Examples
-///
-/// ```
-/// # use vortex_array::IntoArray;
-/// # use vortex_buffer::buffer;
-/// let array = buffer![0_i16, 1, 2, 3, 4].into_array();
-/// assert_eq!(
-///     format!("{}", array.display()),
-///     "[0i16, 1i16, 2i16, 3i16, 4i16]",
-/// )
-/// ```
-///
-/// See also:
-/// [Array::display](http://localhost:8000/doc/vortex_array/trait.Array.html#method.display),
-/// [Array::display_as](http://localhost:8000/doc/vortex_array/trait.Array.html#method.display_as),
-/// [DisplayArrayAs], and [DisplayOptions].
-pub struct DisplayArray<'a>(pub &'a dyn Array);
-
-impl Display for DisplayArray<'_> {
-    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-        self.0.fmt_as(f, &DisplayOptions::default())
-    }
-}
-
 /// Describe how to convert an array to a string.
 ///
 /// See also:
-/// [Array::display](http://localhost:8000/doc/vortex_array/trait.Array.html#method.display),
-/// [Array::display_as](http://localhost:8000/doc/vortex_array/trait.Array.html#method.display_as),
+/// [Array::display_as](./trait.Array.html#method.display_as),
```

Comment:
Have you tested? This is in the display module, would have thought this was ../

Reply:
I'm guessing tbh